README.md

```diff
@@ -0,0 +1,8 @@
+# Alfresco Kubernetes Charts
+
+Source repository for *Alfresco Kubernetes Charts*
```

Comment:
This isn't actually the source, maybe:
> Helm repo for ...

Reply:
Let's also repeat what's in the repo description in this section, except the source part 😄